openfe/tests/storage/conftest.py

```diff
@@ -0,0 +1 @@
+from gufe.tests.conftest import *
```

Comment:
I'm not completely happy with this, it's a bit annoying to have to jump between two packages' conftests (which are already a bit of an opaque pytest magic), but I think we're due a testdata reorganisation in general

Reply:
IMO, we should generally be doing exactly this: reusing `gufe`'s fixtures. Otherwise we're doubling maintenance burden on fixtures (not to mention increasing confusion, since fixtures aren't consistent between the packages). This isn't just about the data, but about the more complicated fixtures that are more than just loading data from a file.